src/main/java/com/kodilla/ecommercee/dto/OrderDto.java

```diff
@@ -11,5 +12,6 @@
     private String status;
     private LocalDateTime dateTime;
     private String shippingAddress;
+    private Cart cart;
```

Comment:
Tutaj raczej DTO dla encji Cart zamiast czysty obiekt encji albo w ogóle sam Long Id Carta z racji OneToOne

Reply:
To jest tylko zaślepka w klasie Seweryna. Wpisałem najprostsze co pozwala skompilować kod. Jak zrobię tutaj Dto to będę musiał w jego mapperze wstrzykiwać mój mapper i zmieniać tam obiekt, myślę że lepiej jak on sobie to zrobi.